WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -1344,6 +1352,19 @@ import Foundation
         case .jetpackSocialTwitterNoticeLinkTapped:
             return "twitter_notice_link_tapped"
 
+        case .jetpackSocialConnectionToggled:
+            return "jetpack_social_auto_sharing_connection_toggled"
+        case .jetpackSocialShareLimitDisplayed:
+            return "jetpack_social_share_limit_displayed"
+        case .jetpackSocialUpgradeLinkTapped:
+            return "jetpack_social_upgrade_link_tapped"
+        case .jetpackSocialNoConnectionCardDisplayed:
+            return "jetpack_social_add_connection_cta_displayed"
+        case .jetpackSocialNoConnectionCTATapped:
+            return "jetpack_social_add_connection_tapped"
+        case .jetpackSocialNoConnectionCardDismissed:
+            return "jetpack_social_add_connection_dismissed"
```

Comment:
This one didn't appear to be in the spec. I've updated the table to include it: pctCYC-PK-p2#analytics. Could you double check it to make sure it's right?

Reply:
Yeah, I noticed that we weren't tracking dismiss events and figured I should add it since the dismiss is permanent. Thanks for adding that to the spec!